pymatgen/io/openmm/tests/test_generators.py

```diff
@@ -73,3 +73,22 @@ def test_get_input_set_w_charges(self):
             "state.xml",
         }
         assert input_set.validate()
+
+    def test_get_input_set_w_charges_and_forcefields(self):
```

Comment:
I think we should add a more diverse set of tests and test more of the possible forcefields. This would be a great case for [pytest parameterize](https://docs.pytest.org/en/6.2.x/parametrize.html), which would let you test a long list of forcefield inputs with almost complete code re-use.

I also think we should make sure that this is actually doing what we expect. Currently, we are just testing to make sure that it doesn't fail, but there is no confirmation that the Systems are actually parameterized differently. We could do this by instantiating a system using `input_set['system.xml'].get_system()` and then using the `System.getForces()` iterator to loop through the forces. I think just looping through all the forces and confirming that they are different for different force fields is enough. I'd recommend testing this with just a lone water molecule and with a lone ethanol (or any organic). That'll keep the resulting `Forces` arrays small. Just generating the `Forces` and then using those values as a test would be sufficient, no need to look up in the Sage or GAFF docs what the values are supposed to be.

Reply:
Sounds great, this will take me a bit longer to get to but at the very least will have it done by next week